samples/core/src/main/resources/db/migration/orcl/V0001__R001_Create_schema_ORCL.sql.orcl

```diff
@@ -0,0 +1,136 @@
+-- Rename this file to V0001__R001_Create_schema_ORCL.sql.orcl if the database used is not Oracle 11g
+
+-- This is the SQL script for setting up the DDL for the h2 database
+-- In a typical project you would only distinguish between main and test for flyway SQLs
+-- However, in this sample application we provde support for multiple databases in parallel
+-- You can simply choose the DB of your choice by setting spring.profiles.active=XXX in config/application.properties
```

Comment:
This is ambiguous: above in file  _samples/core/src/main/resources/application.properties_ you added a comment which says that one has to set the spring profile there. Here you say that one has to set the profile in _config/application.properties_.
I would prefer to set the profile always in the same file, because otherwise some weird dependencies might occur:
Properties are applied depending on the order in which Spring boot traverses application.properties files on start-up. If you have multiple application.properties files which define the same property with different values only the last value is applied. 

Have a lookt at the [spring reference](http://docs.spring.io/spring-boot/docs/1.3.3.RELEASE/reference/htmlsingle/#boot-features-external-config-profile-specific-properties) for fruther information.